TEST ADD (250224@main): [ macOS wk1 Debug ] storage/domstorage/sessionstorage/window-open-remove-item.html is a consistent crash

cdumez opened the pull request:
#### 118ff24f776cfb89088831d8a054156548c6368b
<pre>
TEST ADD (250224@main): [ macOS wk1 Debug ] storage/domstorage/sessionstorage/window-open-remove-item.html is a consistent crash
<a href="https://bugs.webkit.org/show_bug.cgi?id=244338">https://bugs.webkit.org/show_bug.cgi?id=244338</a>
&lt;rdar://99143234&gt;

Reviewed by Sihui Liu.

Make sure we don&apos;t call window.open() before the load event has fired since DRT
doesn&apos;t like that.

* LayoutTests/platform/mac-wk1/TestExpectations:
* LayoutTests/storage/domstorage/sessionstorage/window-open-remove-item.html:

Canonical link: <a href="https://commits.webkit.org/253806@main">https://commits.webkit.org/253806@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/7cd174634e5973b9cbd90e8733638abe03a70471

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/87115 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/69/builds/31202 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/17961 "Built successfully") | [  ~~🛠 wpe~~](https://ews-build.webkit.org/#/builders/8/builds/96110 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 🧪 win](https://ews-build.webkit.org/#/builders/10/builds/149693 "Built successfully and passed tests") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/64/builds/29567 "Built successfully") | [  ~~🛠 mac-debug~~](https://ews-build.webkit.org/#/builders/71/builds/25826 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 gtk~~](https://ews-build.webkit.org/#/builders/36/builds/79247 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/91138 "Built successfully") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/92731 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/68/builds/23877 "Passed tests") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/73931 "Built successfully") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/36/builds/79247 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/78861 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/79124 "Passed tests") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/36/builds/79247 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/66/builds/27294 "Built successfully") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/12961 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/67/builds/27239 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/13975 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/2686 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/63/builds/28923 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/60/builds/36829 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/65/builds/28863 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/33252 "Passed tests") | | 
<!--EWS-Status-Bubble-End-->

szewai reviewed (approved):
LGTM for the change, but the title of the pull request / commit message seems in wrong format?

cdumez committed:
TEST ADD (250224@main): [ macOS wk1 Debug ] storage/domstorage/sessionstorage/window-open-remove-item.html is a consistent crash
https://bugs.webkit.org/show_bug.cgi?id=244338
<rdar://99143234>

Reviewed by Sihui Liu.

Make sure we don't call window.open() before the load event has fired since DRT
doesn't like that.

* LayoutTests/platform/mac-wk1/TestExpectations:
* LayoutTests/storage/domstorage/sessionstorage/window-open-remove-item.html:

Canonical link: https://commits.webkit.org/253806@main
